shared/login/register/paper-key/index.js

```diff
@@ -34,5 +34,5 @@ PaperKey.propTypes = {
 }
 
 export default connect(
-  (state, ownProps) => ownProps.mapStateToProps(state)
+  state => ({waitingForResponse: state.login.waitingForResponse})
```

Comment:
You're right, doesn't work, though we do need to use waitingForResponse to show button spinners -- I'll ask you tomorrow for the right way to get it in a prop.